tlscanary/tools/cache.py

```diff
@@ -46,7 +46,7 @@ def delete(self, id_or_path=None):
         if id_or_path is None:
             self.__clear()
             return
-        path = self[id_or_path.lstrip(self.__root)]
```

Comment:
This was not doing what I believe was intended. It seems like we were hoping that this would left strip an entire string pattern, however the string given to `lstrip` is not a string, per se, but rather a character set to use at a cutset.

For example, it seems like what we *_want_* is...

```python
In [1]: root="/tmp/test/cache"
In [2]: path="/tmp/test/cache/stage_cert_storage"
In [3]: path.lstrip(root)
Out[3]: 'stage_cert_storage'
```

But what we actually get is the much more suprising...

```python
In [1]: root="/tmp/test/cache"
In [2]: path="/tmp/test/cache/stage_cert_storage"
In [3]: path.lstrip(root)
Out[3]: 'ge_cert_storage'
```